### PR TITLE
feat(checkpoints): add `Checkpoints.get` and create Checkpoint entity

### PR DESCRIPTION
**Pull request by monitz87**
Added `totalSupply` and `timestamp` to Checkpoint

### CODE REVIEW

File: src/api/entities/SecurityToken/Checkpoints.ts [hunk 1]
```diff
@@ -24,4 +26,38 @@ export class Checkpoints extends Namespace<SecurityToken> {
...
+      { paginationOpts, arg: rawTicker }
+    );
+
+    const now = new Date();
+    const data = entries
+      .map(([{ args: [, id] }, timestamp]) => ({
+        checkpoint: new Checkpoint({ id: u64ToBigNumber(id), ticker }, context),
+        createdAt: momentToDate(timestamp),
+      }))
+      .filter(({ createdAt }) => createdAt <= now);
```
Comment (VictorVicente) [inline src/api/entities/SecurityToken/Checkpoints.ts:56]: Why do you need these filter?

Comment (monitz87) [reply src/api/entities/SecurityToken/Checkpoints.ts:56]: The query also returns the next scheduled checkpoint's time

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**